Helper/Data.php

```diff
@@ -58,7 +60,21 @@ public function __construct(
         $this->_enc = $enc;
         parent::__construct($context);
     }
-    
+
+    /**
+     * Get Reach Stash API URL
+     *
+     * @return string
+     */
+    public function getStashApiUrl()
+    {
+        if ($this->getConfigValue(self::CONFIG_API_MODE)) {
+            return self::STASH_SANDBOX_URL;
```

Comment:
From the conditional block and what is inside for sure we sense that when the returned value is true we are considering SANDBOX. Checking it against something like SANDBOX_MODE (holding a value equivalent to  true or false) or something equivalent would be cleaner/explicit.

Reply:
I'm afraid I don't really understand your comment. I'm also not the author of this section of code and can't say for sure what is being tested or why. I'm not sure if this is the ticket to make this type of change and perhaps should be isolated to it's own ticket/defect?

Reply:
if ($this->getConfigValue(self::CONFIG_API_MODE) ==  SANDBOX_MODE) something like that